package.json

```diff
@@ -5,7 +5,6 @@
   "repository": "https://github.com/seattleagainstslavery/deterrence-generator",
   "author": "Seattle Against Slavery <info@seattleagainstslavery.org>",
   "dependencies": {
-    "classnames": "^2.2.6",
```

Comment:
🙌 it worked! So glad! This change is from some work of Dennise's, so we'll need that in there too. (the revert you did was to an earlier commit).

Just add back this line and run `npm install` and commit  any changes that come from that

Reply:
Awesome! I have readded the classnames back into the dependecies